third_party/statusor/statusor_test.cc

```diff
@@ -0,0 +1,441 @@
+/**
```

Comment:
I doubt this is going to get run so is it worth copying?

Reply:
I've renamed some methods of the forked StatusOr public API after talking to absl engineer to match what is planned to be open sourced. The file is forked from tensorflow and API names ate outdated. So I needed the test to make sure I did not screw something up. I mean it does not hurt to keep it for now.

Reply:
Sure sounds good.